osu.Game/Graphics/Containers/SectionsContainer.cs

```diff
@@ -189,15 +189,15 @@ protected override void UpdateAfterChildren()
                 headerBackgroundContainer.Height = (ExpandableHeader?.LayoutSize.Y ?? 0) + (FixedHeader?.LayoutSize.Y ?? 0);
                 headerBackgroundContainer.Y = ExpandableHeader?.Y ?? 0;
 
-                T bestMatch = null;
-                float minDiff = float.MaxValue;
+                T bestMatch = Children.FirstOrDefault();
+                float minDiff = float.MinValue;
```

Comment:
This no longer makes sense. This variable was named `minDiff` since it kept the minimum difference between the section position and current scroll position + offset. Now, as far as I can see, it stores the *biggest negative difference* between the two.

Not to mention, as far as I can see this *entire* loop below could be now entirely replaced with

```csharp
SelectedSection.Value = Children.TakeWhile(section => scrollContainer.GetChildPosInContent(section) - currentScroll - scrollOffset < 0)
                                .LastOrDefault();

```

Reply:
Yeah, I didn't know those methods existed...

Reply:
Under the hood it does exactly the same like my loop but whit SelectedSection.Value directly instead of bestMatch